jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ServiceHandlerTest.java

```diff
@@ -105,11 +101,11 @@ public void getServicesWithHeadlessServiceTest() {
         services = serviceHandler.getServices(serviceConfigs);
 
         //Assertion
-        assertFalse(services.isEmpty());
-        assertEquals("NodePort",services.get(0).getSpec().getType());
-        assertEquals("None",services.get(0).getSpec().getClusterIP());
-        assertTrue(services.get(0).getMetadata().getLabels().get("expose").equalsIgnoreCase("true"));
-        assertTrue(services.get(0).getMetadata().getName().equalsIgnoreCase("service-test"));
+        assertThat(services).isNotEmpty();
+        assertThat(services.get(0).getSpec().getType()).isEqualTo("NodePort");
+        assertThat(services.get(0).getSpec().getClusterIP()).isEqualTo("None");
+        assertThat(services.get(0).getMetadata().getLabels().get("expose")).isEqualToIgnoringCase("true");
+        assertThat(services.get(0).getMetadata().getName()).isEqualToIgnoringCase("service-test");
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.